template/php7/readme.md

```diff
@@ -0,0 +1,37 @@
+# PHP7 Template
+
+Templates are built using the latest minor version of the major release.
+
+| Modules (by default) |
+| ------------- |
+| Core, date, libxml, openssl, pcre, sqlite3, zlib, ctype, curl, dom, fileinfo, filter, ftp, hash, iconv, json, mbstring, SPL, PDO, pdo_sqlite, session, posix, readline, Reflection, standard, SimpleXML, Phar, tokenizer, xml, xmlreader, xmlwriter, mysqlnd, sodium |
+
+## Usage:
+
+```shell
+faas-cli new my-function-php7 --lang php7
```

Comment:
Super picky thing, but I'd drop `-php7` from the function name just in case someone might misinterpret that as needed.